config-static/README.md

```diff
@@ -0,0 +1,35 @@
+# Wodin Static
+
+## Background
+
+We would like researchers to be able to build a site without a server and publish a static site somewhere like github pages. This will be to build interactive research papers/articles that allow policy makers to better understand a given model. Since the model will be static we can remove both the `express` backend, `redis` and `odin.api`.
+
+## Run/Development
+
+An example of the config a user needs to design is in this folder. To convert this example into a site run:
+1. `./scripts/run-dev-dependencies.sh`
+1. `./scripts/build-and-serve-static-site.sh`
+The site should be available at `localhost:3000` (the run dependencies script is only involved in generating code for this site. Once the site is built, you can turn these off and still change the HTML document, e.g. adding a new graph, and it'll update everything accordingly when you refresh the page).
+
+## How it works
+
+There are two stages to this process, build time (code and static asset generation we do in a github action once the user has created a config) and run time (querying the HTML page and mounting the correct components with the correct stores).
+
+### Build time
+
+After the user writes the config, a github action will checkout this repo, run the `odin.api` docker container and run the `wodinBuilder.ts` script which takes in the path of the config and path of the output folder as args. This script expects a `stores` folder which contains folders that label your stores, `<store-name>`. In each `<store-name>` folder it expects a `config.json` and `model.R` file. This script does a couple of things:
+1. Copies the `index.html` file the user has written to the output folder
+1. Gets the Javascript code for the ODE and discrete runners and saves them into `stores/runnerOde.js` and `stores/runnerDiscrete.js` (the runner are generic and shared across all stores)
+1. For each `<store-name>`, it saves the config into `stores/<store-name>/config.json` and it compiles the model code and saves the response into `stores/<store-name>/model.json`.
+
+We also build `wodin` frontend in static mode (just normal `wodin` frontend with an early return in the api service so we don't do any network requests) and copy the js and css files into the output folder. After this, we just commit the output folder into a github pages branch and github will deploy it.
```

Comment:
Worth saying here why we need to do early returns from the api service, and why it's benign that we do..?

Will the github pages bit be part of the main wodin build gha? It's implied here. We could make it an optional setting to do this - that would make things really easy for the user, but won't always be wanted if not using ghp.